test/base/module/bulkRequest/BulkRequest.addRecords.test.js

```diff
@@ -0,0 +1,16 @@
+import Auth from '../../../../src/node/authentication/Auth';
+import Connection from '../../../../src/node/connection/Connection';
+import BulkRequest from '../../../../src/base/module/bulkRequest/BulkRequest';
+import {USERNAME, PASSWORD, DOMAIN} from './common';
+
+describe('Checking BulkRequest.addRecords function', () => {
+  it('can be called without param', () => {
+    const auth = new Auth();
+    auth.setPasswordAuth({username: USERNAME, password: PASSWORD});
+    const connection = new Connection({auth, domain: DOMAIN});
+    const bulkRequest = new BulkRequest({connection});
+    expect(bulkRequest).toBeInstanceOf(BulkRequest);
+    const result = bulkRequest.addRecords();
```

Comment:
@phongnhat19 
const result = bulkRequest.addRecords();
expect(result).toBeInstanceOf(BulkRequest);
=>
bulkRequest.addRecords();
//expect more after addRecords without param called an empty array of requests is added with empty/undefined record